code/game/objects/structures/dresser.dm

```diff
@@ -56,27 +56,22 @@
 
 /obj/structure/dresser/crowbar_act(mob/user, obj/item/I)
 	. = TRUE
-	if(!I.tool_start_check(src, user, 0))
+	if(!I.use_tool(src, user, 0))
 		return
 	TOOL_ATTEMPT_DISMANTLE_MESSAGE
 	if(I.use_tool(src, user, 50, volume = I.tool_volume))
 		TOOL_DISMANTLE_SUCCESS_MESSAGE
-
+		deconstruct(disassembled = TRUE)
 
 /obj/structure/dresser/wrench_act(mob/user, obj/item/I)
 	. = TRUE
-	if(!I.use_tool(src, user, 0, volume = I.tool_volume))
+	if(!I.tool_use_check(user, 0))
```

Comment:
This one is already done in `default_unfasten_wrench`

Reply:
Didn't really look at the code for `default_unfasten_wrench`, fixed now.